.github/workflows/integration.yml

```diff
@@ -28,9 +28,13 @@ jobs:
               GITHUB_HEAD_REF=${GITHUB_REF##*/}
           fi
           GIT_COMMIT=$(git rev-parse HEAD | head -c 8)
-          REMOTE="https://github.com/${{ github.repository }}"
+          REMOTE=${{ github.event.pull_request.head.repo.html_url }}
```

Comment:
I'm confused about why this is showing up in the diff. The current state of master looks like the right hand side, not the left: 

https://github.com/ethereum-optimism/contracts/blob/master/.github/workflows/integration.yml#L31-L37